dev-docs/adunit-reference.md

```diff
@@ -0,0 +1,342 @@
+| `native` | Required, unless either of the other properties are present. | Object | Defines properties of a native ad.  For properties, see [`adUnit.mediaTypes.native`](#native). |
+
+<a name="banner" />
+
+#### `adUnit.mediaTypes.banner`
+
+{: .table .table-bordered .table-striped }
+| Name    | Scope    | Type                                  | Description                                                                             |
+|---------+----------+---------------------------------------+-----------------------------------------------------------------------------------------|
+| `sizes` | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`. |
+| `name`  | Optional | String                                | Name for this banner ad unit.  Can be used for testing and debugging.                   |
+
+<a name="native" />
+
+#### `adUnit.mediaTypes.native`
+
+The `native` object contains the following properties that correspond to the assets of the native ad.
+
+{: .table .table-bordered .table-striped }
+| Name          | Scope    | Type                                  | Description                                                                                                                       |
+|---------------+----------+---------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------|
+| `sizes`       | Required | Array[Number] or Array[Array[Number]] | All sizes this ad unit can accept.  Examples: `[400, 600]`, `[[300, 250], [300, 600]]`.                                           |
+| `type`        | Optional | String                                | Used as a shorthand, e.g., `type: 'image'` implies required fields `image`, `title`, `sponsoredBy`, `clickUrl`.                   |
+| `title`       | Required | Object                                | The title of the ad, usually a call to action or a brand name.  For properties, see [`native.title`](#native-title).              |
+| `body`        | Required | Object                                | Text of the ad copy.  For properties, see [`native.body`](#native-body).                                                          |
+| `sponsoredBy` | Required | String                                | The name of the brand associated with the ad.  For properties, see [`native.sponsoredBy`](#native-sponsoredby).                   |
+| `icon`        | Optional | Object                                | The brand icon that will appear with the ad.  For properties, see [`native.icon`](#native-icon).                                  |
+| `image`       | Optional | Object                                | A picture that is associated with the brand, or grabs the user's attention.  For properties, see [`native.image`](#native-image). |
```

Comment:
"The image object is to be used for the main image of the Native ad."

Reply:
Updated in 27bc64b